dipy/workflows/viz.py

```diff
@@ -24,7 +24,15 @@ class HorizonFlow(Workflow):
     @classmethod
     def get_short_name(cls):
         return 'horizon'
-
+    
+    def manage_output_overwrite_helper(self):
+        '''
+        Overriding parent function to stump warnings of over writing existing
+        files as interactive mode of HorizonFlow doesn't attempt to write any 
+        output files
+        '''
+        return True
```

Comment:
Hi @lb-97,

This is not a solution to overwrite this overwrite the helper. 

Please, look deeper to check why it is needed

Reply:
Hi @skoudoro,
Here's a summary of my understanding: _manage_output_overwrite_, is a native method of class Workflow and HorizonFlow is inherited from this class. The method above is responsible for generating an iterator of tuples of input, output files(to be generated). The output file names are identified by the arguments 'out_*', the default value of these variables i.e., 'tmp.png' is considered if nothing is explicitly passed. Now, this method generators an empty iterator if 'tmp.png' already exists by throwing a WARN message. 

In the interactive mode of HorizonFlow, the default 'out_*' argument is picked up by the parent method, although not desired and returns an empty iterator with a WARN message. When the iterator is empty, no input files can be visualized. 

The parent _manage_output_overwrite_ method is used at many other places in the code, so one way to generate an iterator but also override WARN message, which is how interactive mode should work, is to make change in the inherited class. That's exactly what my PR does. 

I believe I made this decision after careful consideration and looking deeper for hours before making a PR.

So, if there's a better way to do it I'm all ears for it. Looking forward.

Reply:
- why using `tmp.png` and not `None`?
- why not just initialize `HorizonFlow` with `self._force_overwrite`?
- With your solution, you hide any overwrite which seems to not be desired behavior ?
- Are you sure that "HorizonFlow doesn't attempt to write any output files in interactive mode" ? 

Also, please:
- add tests to reproduce the issue and show the correction
- follow the library coding style (double quotes instead of single quotes for example)

all this note should help you to go deeper into it and follow the library standard.

Reply:
Hi @skoudoro ,
1. `tmp.png` is the default filename used in stealth mode. It's the expected name in other examples such as `save_tractogram()`.  Adding the below change does work but it's more of a hack than understanding root cause. My PR addresses the root cause of this warning, as well as override it.
`if interactive:
    out_stealth_png = ''
`
2. Initializing with `self._force_overwrite`, will output the message:
`The following output files are about to be overwritten.`
Although it's actually not overriding anything
3. The expected behavior of interactive mode of HorizonFlow is to open an interactive window but not save/override any file. I verified this by implementing it locally.
4. Yes.

Sure, I'd be happy to add any test cases. Since this is not a New Feature type issue, I am not sure what kind of new test cases I can introduce. 
Yes, I will change it to double quotes, my bad. I apologize.

Looking forward to hearing from you.